api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/standalone/StandaloneSecurityConfig.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.apicatalog.standalone;
+
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@ConditionalOnProperty(value = "apiml.catalog.standalone.enabled", havingValue = "true")
+public class StandaloneSecurityConfig {
+
+    @Bean
+    @Order(Ordered.HIGHEST_PRECEDENCE)
+    public SecurityFilterChain permitAll(HttpSecurity http) throws Exception {
+        return http
```

Comment:
could you please add warn log that the standalone mode is enabled?

Reply:
What do you think about adding an info message? Since it is expected behavior in standalone mode, admins could think something is wrong.

Reply:
it is expected when admins want to use standalone mode, but what if someone turns this on by accident? Or they don't fully understand standalone mode.

Reply:
There will be no other public documentation except the Github repo. In any case, Catalog is disconnected from GW and DS in standalone mode. So it can't show any sensitive data. But adding the info message that Standalone mode is active is a  good idea.

Reply:
There's probably no use case for the standalone catalog in the mainframe, right? Is it ok then to show it as an info message? Or if it needs to be a warning one, could it be warning only when running on z/OS?

Reply:
Showing a warning when running on zos could be a good compromise. But let's wait for Andrej's opinion.

Reply:
After discussing with Andrej we added a warn message